File: Pr10.java
import java.util.Arrays;
import java.util.Scanner;
public class Pr10 {
        public static boolean areAnagrams(String str1, String str2) {
        
        str1 = str1.replaceAll("\\s", "").toLowerCase();
        str2 = str2.replaceAll("\\s", "").toLowerCase();

        if (str1.length() != str2.length()) {
            return false;
        }

        char[] charArray1 = str1.toCharArray();
        char[] charArray2 = str2.toCharArray();
        Arrays.sort(charArray1);
        Arrays.sort(charArray2);

      
        return Arrays.equals(charArray1, charArray2);
    }

    public static void main(String[] args) {
        String str1;
        String str2;
        
        Scanner inp = new Scanner(System.in);
        System.out.print("Enter a first string: ");
        str1=inp.nextLine();
        System.out.print("Enter a second string: ");
        str2=inp.nextLine();

        if (areAnagrams(str1, str2)) {
            System.out.println("The strings are anagrams.");
        } else {
            System.out.println("The strings are not anagrams.");
        }
    }
}

File: Assignment_1_cpp/Pr10.cpp
#include <iostream>
#include <string.h>
using namespace std;

int sort(char s[])
{
	int i,j,n,temp;
	n = strlen(s);
	for(i=0;i<n-1;i++)
	{
		for(j=i+1;j<n;j++)
		{
			if(s[i]>s[j])
			{
				temp = s[i];
				s[i] = s[j];
				s[j] = temp;
			}
		}
	}
}
int main()
{
	char s1[100],s2[100];
	
	cout<<"Enter first string: ";
	cin>>s1;
	cout<<"Enter second string: ";
	cin>>s2;
	
	sort(s1);
	sort(s2);
	
	if(strcmp(s1,s2)==0)
	{
		cout<<endl<<"True"<<endl;
		cout<<"Given String are anagrams!";
	}
	else
	{
		cout<<endl<<"False"<<endl;
		cout<<"Given String are not anagrams!";
	}
}